### DISCUSSION TITLE
Optional parameter $force declared before required parameter $testType is implicitly treated as a required parameter

### DISCUSSION THREAD

**Discussion by nishpatel** [livewire/livewire#7315] [7. Bugs (archived)]
### Livewire version

v3.2.0

### Which PHP version are you using?

PHP 8.2

### Repository URL

https://github.com/livewire/livewire/pull/7248/files

### Steps To Reproduce

composer update

### Additional information

Bug introduced by:

https://github.com/livewire/livewire/pull/7248/files

protected function createTest($force = false, $testType)

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@nishpatel thanks for reporting! I have submitted a fix PR #7319
